geps/gep-3949/index.md

```diff
@@ -0,0 +1,459 @@
+# GEP-3949: Mesh Resource
```

Comment:
Overall, I do not think this is a net benefit to the project. That being said, the downside if I am right about that is minimal -- even in the worst case, this likely is a minor cruft that no one cares about rather than a major issue.

* As previously expressed, I feel putting documentation (in the form of `supportedFeatures`) is an inappropriate usage of Status.
* I also feel `parametersReference` on an object that is auto-generated for users (and cluster scoped) is also problematic.
  * Aside from the above issue, I don't think Istio would use this field for other reasons. Which is fine, just calling it out.

Fully acknowledging this seems to be popular for others so happy to agree to disagree

Reply:
Forgot to actually answer the question. I don't think it changes things too much. I'm struggling to see a way this resource benefits Istio users which is perfectly fine especially if we can reduce ways it confuses or otherwise harms the users, which is already low-risk and plausibly even better as we are ironing out the install stuff, but also doesn't make me in favor of this (at best, ambivalent)

@keithmattix any areas you think this would help (or hurt) that I may be missing?

Reply:
Resolving this one like I should've done last night. 🤦‍♂️